components/automate-ui/src/app/pages/data-feed-create/data-feed-create.component.ts

```diff
@@ -236,7 +285,8 @@ export class DataFeedCreateComponent {
     switch (this.integTitle) {
       case WebhookIntegrationTypes.SERVICENOW:
       case WebhookIntegrationTypes.SPLUNK:
-      case WebhookIntegrationTypes.ELK_KIBANA: {
+      case WebhookIntegrationTypes.ELK_KIBANA:
+      case WebhookIntegrationTypes.CUSTOM: {
```

Comment:
Can we alse check if header checkbox is checked and header input is not empty?  Also i feel we should validate whats entered in the header here itself, what do you think ?

Reply:
We are checking header is empty or not under Authtype case.